app/src/main/java/br/com/concrete/tentacle/features/loadmygames/LoadMyGamesViewHolder.kt

```diff
@@ -0,0 +1,21 @@
+package br.com.concrete.tentacle.features.loadmygames
+
+import android.widget.LinearLayout
+import androidx.recyclerview.widget.RecyclerView
+import br.com.concrete.tentacle.data.models.Media
+import kotlinx.android.synthetic.main.fragment_game_item.view.*
+
+class LoadMyGamesViewHolder(
+    private val mLinearLayout: LinearLayout
+) : RecyclerView.ViewHolder(mLinearLayout) {
+
+    companion object {
+        fun <T> callBack(holder: RecyclerView.ViewHolder, element: T) {
+
+            val gameViewHolder = holder as LoadMyGamesViewHolder
```

Comment:
acho que aqui também seria interessante tu colocar uma verificação. 
`if(holder is LoadMyGamesViewHolder) {.......}`